### PR TITLE
Add boilerplate

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Add boilerplate to all plugins and unit tests that were missing it.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
collection

### CODE REVIEW

**Comment by felixfontein** [comment]
CC @Akasurde @Andersson007

File: tests/unit/plugins/modules/network/nso/nso_module.py [hunk 1]
```diff
@@ -15,7 +15,8 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-from __future__ import (absolute_import, division, print_function)
+from __future__ import absolute_import, division, print_function
```
Comment (Andersson007) [inline tests/unit/plugins/modules/network/nso/nso_module.py:18]:
```suggestion
from __future__ import absolute_import, division, print_function
```
earlier you pasted this line with parentheses as well:)

**Review by Andersson007 — APPROVED** [review]
LGTM

File: tests/unit/plugins/modules/network/nso/nso_module.py (hunk 1 above)
Comment (felixfontein) [reply tests/unit/plugins/modules/network/nso/nso_module.py:18]: I usually copied them from similar files :)

**Comment by felixfontein** [comment]
@Andersson007 thanks a lot for reviewing this!

**Pull request merged**